scripts/release.js

```diff
@@ -156,6 +156,21 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install component and sdk and build the test app to ensure no peer dependency warnings, errors or build issues
+logger.log('Create a new npm package and install component and js sdk as dependencies, check if there is peer dependency warning for component library');
```

Comment:
Would it be better to write down the full name of the repositories? e.g

```
logger.log(
    'Create a new NPM package and install amazon-chime-sdk-component-library-react 
    and amazon-chime-sdk-js as dependencies, check if there is peer dependency warning for 
    amazon-chime-sdk-component-library-react'
);
```

Reply:
I like the idea. will add. Thanks!